core/src/edu/asu/c3simulator/screens/DifficultySelectionScreen.java

```diff
@@ -185,5 +192,5 @@ public void show()
 	{
 		Gdx.input.setInputProcessor(stage);
 	}
-	
+
 }

```

Comment:
I'm not sure what your settings are for line width or format, but I am just using the default.